app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/minigame/FireOutbreakActivity.java

```diff
@@ -168,5 +179,4 @@ public void onPlasmaPress(View view) {
             validateAction(-3);
```

Comment:
wouldn't it be better to just use one number for the else cases, instead of -1,-2,-3?